src/components/connection/AddDatabaseForm.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div class="save-connection expand">
+    <h3 class="dialog-c-title">Add database</h3>
+
+    <div v-if="error" class="alert alert-danger">
+      {{error}}
+    </div>
+
+    <div class="form-group">
+      <input class="form-control" v-model="databaseName" @keydown.enter.prevent.stop="save" type="text" placeholder="Database Name">
```

Comment:
So this keydown does work, but the right way to do this is to have a `<form>` tag with a `submit` event.

Then the `cancel` button has `type=button`, and you don't need any keydown on the input. For a good example see `ImportButton.vue`